google/cloud/datacatalog/v1beta1/datacatalog_gapic.yaml

```diff
@@ -127,6 +127,33 @@ interfaces:
     retry_codes_name: non_idempotent
     retry_params_name: default
     timeout_millis: 60000
+    samples:
+      standalone:
+      - value_sets: [data_catalog_search]
+        region_tag: data_catalog_search
+    sample_value_sets:
+    - id: data_catalog_search
+      description: Search Catalog
+      parameters:
+        defaults:
+        - scope.include_project_ids[0] = "[Google Cloud Project ID]"
+        - scope.include_gcp_public_datasets = false
+        - query = "[String in search query syntax]"
```

Comment:
Could we get a hint as to what a typical search query looks like? Or maybe a comment with a link to the description of the syntax?